### ISSUE TITLE
[REQUEST] New block creation after selected block

### ISSUE THREAD

**Issue by dNitro** [openbullet/openbullet#427]
**Is your feature request related to a problem? Please describe.**
An issue with big configs is that new block creation always goes at end of the queue and someone should constantly click up to move it to the desired position ( or move it loliScript )

**Describe the solution you'd like**
It would be nice if the new block positioned right after the selected block. This way when we select a block and create a new block ( + ) it right goes to the position and doest need to move.

**Comment by Unkn9wn1** [comment]
it's not a good idea bro
this will make more problems
if you add ability to move BLOCKS using mouse, will solve the problem in a better way and will not make more problems
@openbullet

**Comment by Unkn9wn1** [comment]
shortcuts, like [Ctrl + Arrows] / [Ctrl + Moving block using mouse left click] will solve this

**Comment by openbullet** [comment]
Sounds nice, I will try to add blocks reordering through drag and drop.

**Comment by openbullet** [comment]
For now the first solution will be adopted, I will look into drag and drop after 1.2.1 since it's not very easy to implement properly and requires quite some time to code.

**Comment by dNitro** [comment]
@Unkn9wn1, Of course, the ultimate solution to this is *drag and drop* and ability to multi-select blocks using meta keys ( CTRL, SHIFT, ... ). But for now, we could have this simple solution till there as @openbullet mentioned ^-^

**Comment by Unkn9wn1** [comment]
@openbullet 
please roll-back this or add option to disable this
now i have to click on last block before creating new block, every time!
or when i not changed block names yet-
i have to keep looking to find where my new block is
fixed Location for new created blocks is better
u can add drag and drop shortcuts [ctrl + mouse LC] instead of this


1.2.1#637

**Comment by Unkn9wn1** [comment]
![image](https://user-images.githubusercontent.com/41063376/78779624-7c9c9e80-79b2-11ea-929b-876acb2b1c2e.png)
if it's temporary, no problem
just please roll-back it after adding drag and drop

also drag and drop without 'drag handle' or while not pressing other key like 'Ctrl', only with a mouse click and moving, will make more problems like this one, users could break their config with just a mouse click and they should keep looking to find what is the problem of their config
